archive/g/go/testinfo.yml

```diff
@@ -0,0 +1,8 @@
+folder:
+  extension: ".go"
+  naming: "hyphen"
+
+container:
+  image: "golang"
+  tag: "1"
```

Comment:
Go has opted to do versioning as x.y... where every change in y is a major release, so let's make the tag 1.12 to be safe.

Reply:
Ah I see. The Version 1 image was updated 6 days, so I kind of lazily selected that for simplicity.